msbuild/Xamarin.MacDev.Tasks.Core/Tasks/CompileAppManifestTaskBase.cs

```diff
@@ -158,16 +172,25 @@ bool SetMinimumOSVersion (PDictionary plist)
 			}
 
 			if (string.IsNullOrEmpty (minimumOSVersionInManifest)) {
-				minimumOSVersion = SdkVersion;
+				// Nothing is specified in the Info.plist - use SupportedOSPlatformVersion, and if that's not set, then use the sdkVersion
+				if (!string.IsNullOrEmpty (convertedSupportedOSPlatformVersion)) {
+					minimumOSVersion = convertedSupportedOSPlatformVersion;
+				} else {
+					minimumOSVersion = SdkVersion;
+				}
 			} else if (!IAppleSdkVersion_Extensions.TryParse (minimumOSVersionInManifest, out var _)) {
 				Log.LogError (null, null, null, AppManifest, 0, 0, 0, 0, MSBStrings.E0011, minimumOSVersionInManifest);
 				return false;
+			} else if (!string.IsNullOrEmpty (convertedSupportedOSPlatformVersion) && convertedSupportedOSPlatformVersion != minimumOSVersionInManifest) {
+				// SupportedOSPlatformVersion and the value in the Info.plist are not the same. This is an error.
+				Log.LogError (null, null, null, AppManifest, 0, 0, 0, 0, "The {0} value in the Info.plist ({1}) does not match the SupportedOSPlatformVersion value in the project file ({2}).", minimumVersionKey, minimumOSVersionInManifest, SupportedOSPlatformVersion); // FIXME: add tests + translated string.
```

Comment:
`// FIXME: add tests + translated string.`

Reply:
🤦

Reply:
@spouliot fixed now (I'd already written the test, so I just had to localize the message).